api/controllers/game/load-fixture.js

```diff
@@ -112,8 +112,16 @@ module.exports = function (req, res) {
 
       return Promise.all([game, ...updatePromises]);
     })
-    .then(function populateGame(values) {
+    .then(async function removeCardsFromDeck(values) {
       const [game] = values;
+      // If deck was specified, delete all other cards from the deck
+      const {deck} = req.body;
+      if (deck) {
+        await Game.replaceCollection(game.id, 'deck').members(deck);
```

Comment:
So it kinda stinks to make this `async` in the middle of a `.then()` promise chain. I think ultimately the whole endpoint should be rewritten to use `async/await`, but I wanted to try to limit the scope of this pr.

The main reason that I wanted async here is that it works nicely for a conditional update like we have below. I only want to overwrite the `deck` if the fixture explicitly specifies it. imo that's a nice use case for `await`, since including it otherwise would require a `Promise.all()` and conditionally pushing an extra update promise to the returned result. I think this is better for now, though I don't love it

Reply:
I don't see anything wrong with it, refactor the rest later.